application/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
 		    using (var amz = new AmazonSynchronizer(new Configuration()))
 		    {
-                amz.WithInterval(TimeSpan.FromMinutes(1))
+                amz.WithInterval(TimeSpan.FromMinutes(5))
```

Comment:
no this is the time between two full database updates, i.e : going through all the products of the database and updating them.

it would be unwise for the system to go through the update of the AmazonProduct table and when it finishes it it restarts again immediately, because price/qty won't change that fast.

At the moment the full update takes around 45 min.

The time between two Amazon requests is set in AmazonSynchronizer.ExecuteAndWait

delayInMs = 1000 ==> 1000ms = 1 sec